pkg/cmd/pnl.go

```diff
@@ -21,6 +21,7 @@ import (
 func init() {
 	PnLCmd.Flags().String("exchange", "", "target exchange")
 	PnLCmd.Flags().String("symbol", "BTCUSDT", "trading symbol")
+	PnLCmd.Flags().Int("limit", 500, "number of orders")
```

Comment:
I would like to know the purpose of this flag, could we set the default to 0? because we should have all trades to be calculated to get the correct result.

Reply:
I think the pnl should be calculated in a specified timeframe. However, the trade count limit is a alternative method to set the timeframe if the user has some knowledge on databases.

Maybe we can improve it to an more user-friendly way.

Reply:
ok, sounds great